weave/panels/panel_facet.py

```diff
@@ -14,6 +14,8 @@ class FacetDimsConfig:
     y: str
     select: str
     detail: str
+    xAxisLabel: str = ""
```

Comment:
please follow the convention of the other properties and make these nodes, not strings